Update kubernetes-walkthrough.md

fredrikhaglund opened the pull request:
Add additional commands needed to succeed with walkthrough.
Add link to information in what locations preview is available.

azurecla commented:
Hi __@JakobGSvendsen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@JakobGSvendsen Thanks for the contribution!  I've sent the change to the author for approval.

bwren commented:
Great addition, thanks.  Can we reword to something like the following to make it a bit more clear?

An input parameter can be accessed in an activity by using a PowerShell Expression data source for a parameter value.  Use the name of the input variable for the expression.  For example, if you specified StringParam for the name of the input variable, then specify $StringParam for the expression.

JakobGSvendsen commented:
Yes please. Thanks!  :)

Sent from my Sony Xperia™ smartphone


---- Brian Wren wrote ----


Great addition, thanks. Can we reword to something like the following to make it a bit more clear?

An input parameter can be accessed in an activity by using a PowerShell Expression data source for a parameter value. Use the name of the input variable for the expression. For example, if you specified StringParam for the name of the input variable, then specify $StringParam for the expression.

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-content/pull/3807#issuecomment-114286125>.

bwren commented:
I don't think I can make that change. Think you need to change the text and resubmit.

JakobGSvendsen commented:
ok i think I got it now :D great feature this is!

deneha commented:
Fixed merge conflict and incorporated edits into https://github.com/Azure/azure-content-pr/pull/12260

fredrikhaglund committed:
Update kubernetes-walkthrough.md

Add additional commands needed to succeed with walkthrough.
Add link to information in what locations preview is available.
